tests/test_overpass.py

```diff
@@ -0,0 +1,145 @@
+from unittest import TestCase
+
+from process_subways import compose_overpass_request
+
+
+class TestOverpassQuery(TestCase):
+    def test__compose_overpass_request__no_bboxes(self) -> None:
+        bboxes = []
+        for overground in (True, False):
+            with self.subTest(msg=f"{overground=}"):
+                with self.assertRaises(RuntimeError):
+                    compose_overpass_request(overground, bboxes)
+
+    def test__compose_overpass_request__one_bbox(self):
+        bboxes = [[1, 2, 3, 4]]
+
+        expected = {
+            False: """
+[out:json][timeout:1000];
+(
+  (
+    rel[route="light_rail"](1,2,3,4);
+    rel[route="monorail"](1,2,3,4);
+    rel[route="subway"](1,2,3,4);
+    rel[route="train"](1,2,3,4);
+  );
+  rel(br)[type=route_master];
+  node[railway=subway_entrance](1,2,3,4);
+  node[railway=train_station_entrance](1,2,3,4);
+  rel[public_transport=stop_area](1,2,3,4);
+  rel(br)[type=public_transport][public_transport=stop_area_group];
+);
+(._;>>;);
+""".replace(
```

Comment:
EZ
```python
        expected = {
            False: (
                "[out:json][timeout:1000];"
                "("
                "("
                'rel[route="light_rail"](1,2,3,4);'
                'rel[route="monorail"](1,2,3,4);'
                'rel[route="subway"](1,2,3,4);'
                'rel[route="train"](1,2,3,4);'
                ");"
                "rel(br)[type=route_master];"
                "node[railway=subway_entrance](1,2,3,4);"
                "node[railway=train_station_entrance](1,2,3,4);"
                "rel[public_transport=stop_area](1,2,3,4);"
                "rel(br)[type=public_transport]"
                "[public_transport=stop_area_group];"
                ");"
                "(._;>>;);"
                "out body center qt;"
            ),
            True: (
                "[out:json][timeout:1000];"
                "("
                "("
                'rel[route="aerialway"](1,2,3,4);'
                'rel[route="bus"](1,2,3,4);'
                'rel[route="ferry"](1,2,3,4);'
                'rel[route="tram"](1,2,3,4);'
                'rel[route="trolleybus"](1,2,3,4);'
                ");"
                "rel(br)[type=route_master];"
                "rel[public_transport=stop_area](1,2,3,4);"
                "rel(br)[type=public_transport]"
                "[public_transport=stop_area_group];"
                ");"
                "(._;>>;);"
                "out body center qt;"
            ),
        }
```

Reply:
This layout doesn't allow copying the query to Overpass Turbo etc. without significant effort . But ok, I've accepted the suggestion.

Reply:
But this is just data for testing, why one will copy it to Overpass?

Reply:
Maybe for debugging tests...
Another point. The data for testing should be maintained like other code, and if the overpass query will change, like in this PR, it should be corrected accordingly. It would be harder now, when it contains many single and double quotes and has no proper indentation.

I think this is not a topic for deeper debate. Can we get closer to merging?